run_single_spark.sh

```diff
@@ -0,0 +1,122 @@
+${SPARK_HOME}/bin/spark-submit                                                                                              \
+    --class  com.databricks.spark.sql.perf.RunTPCBenchmark                                                                  \
+    --conf  spark.kryo.referenceTracking=true                                                                               \
+    --conf  spark.kryoserializer.buffer.max=256m                                                                            \
+    --conf spark.shuffle.io.numConnectionsPerPeer=4                                                                         \
+    --conf spark.reducer.maxSizeInFlight=128m                                                                               \
+    --conf spark.executor.extraJavaOptions="-Diop.version=4.1.0.0 -XX:ParallelGCThreads=${gcThreads} -XX:+AlwaysTenure ${executor_extraJavaOptions}"      \
+    ${extraOptions}                                                                                                         \
+    --conf spark.sql.shuffle.partitions=${sql_shuffle_partitions}                                                           \
+    --conf spark.yarn.driver.memoryOverhead=400                                                                             \
+    --conf spark.yarn.executor.memoryOverhead=${executor_memoryOverhead}                                                    \
+    --conf spark.shuffle.consolidateFiles=true                                                                              \
+    --conf spark.reducer.maxSizeInFlight=128m                                                                               \
+    --conf spark.sql.autoBroadcastJoinThreshold=67108864                                                                    \
+    --conf spark.serializer=org.apache.spark.serializer.KryoSerializer                                                      \
+    --master spark://sparkhab1lab.austin.ibm.com:7077                                                                       \
```

Comment:
@lybrian, How is this file any different from `run_single.sh` ?

I see only the master input parameter is replaced. For this, you can modify the `run_single.sh` to take extra parameters.

Kindly don't hardcode any values like `sparkhab1.lab..` into the scripts. We will try to keep as generic as possible and usable by multiple teams.